### PR TITLE
Restucturing the response

### PR DESCRIPTION
**Pull request by nawodbiz**

### CODE REVIEW

File: Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/JsonStringService.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+@Service
+public class JsonStringService {
+    String jsonString1 ="";
+    String jsonString2 ="";
+
+
+
+    List<String> longDuration = new ArrayList<>();
+
```
Comment (krishanth-nimi) [inline Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/JsonStringService.java]: Remove unnecessary whitespaces

File: Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/JsonStringService.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    public JSONObject getShortDuration2 (String duration){
+
+
+        Pattern pattern = Pattern.compile("([A-Za-z]*)\\s(\\d*)");
+        Matcher matcher = pattern.matcher(duration);
+        Boolean matchFound = matcher.find();
+
+        if(matchFound) {
+
+
```
Comment (krishanth-nimi) [inline Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/JsonStringService.java]: Remove unnecessary whitespaces

File: Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/JsonStringService.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            JSONObject jsonObject = new JSONObject();
+                   jsonObject.put("month", matcher.group(1));
+                    jsonObject.put("year", matcher.group(2));
+
+
+
+            return jsonObject;
+
```
Comment (krishanth-nimi) [inline Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/JsonStringService.java]: Remove unnecessary whitespaces

File: Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java [hunk 4]
```diff
@@ -63,6 +72,23 @@ public String extractExperiences(MultipartFile file) throws IOException {
     String timePeriod = "";
     String location= "";
 
+//    String duration = "";
```
Comment (krishanth-nimi) [inline Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java]: Remove unwanted comments

File: Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java [hunk 5]
```diff
@@ -169,8 +195,17 @@ public String extractExperiences(MultipartFile file) throws IOException {
 
 
                 if(matchFound){
-                    timePeriod = matcher.group(0);
+                    timePeriod = matcher.group(1);
+
+//                    longDuration = jsonStringService.getLongDuration(timePeriod);
```
Comment (krishanth-nimi) [inline Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java]: Remove unwanted comments

File: Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java [hunk 6]
```diff
@@ -218,14 +252,69 @@ public String extractExperiences(MultipartFile file) throws IOException {
             experienceList2.add(timePeriod);
```
Comment (krishanth-nimi) [inline Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java]: Use meaningful names for the variables

File: Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java [hunk 7]
```diff
@@ -218,14 +252,69 @@ public String extractExperiences(MultipartFile file) throws IOException {
...
+
+                JSONObject jsonString = new JSONObject();
+                        jsonString.put("company", company);
+                        jsonString.put("title", title);
+                        jsonString.put("time duration", jsonStringService.getLongDuration(timePeriod));
+                        jsonString.put("description", description);
+
+                positionsList.add(jsonString);
+
+
```
Comment (krishanth-nimi) [inline Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java]: Remove unwanted whitespaces

File: Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java [hunk 8]
```diff
@@ -300,13 +407,13 @@ public String extractExperiences(MultipartFile file) throws IOException {
             experienceList2.add(timePeriod);
```
Comment (krishanth-nimi) [inline Project-Resume-Graph/src/main/java/com/example/Project/Resume/Graph/service/ReadPdfService.java]: Use meaningful names for the variables

**Review by nawodbiz — COMMENTED** [review]
code resolved as the guidelines

**Pull request closed without merging**